docs/examples.rst

```diff
@@ -688,8 +688,8 @@ Inject the middleware into the middleware onion
 
 .. code-block:: python
 
-    from web3.middleware import extradata_to_poa_middleware
-    w3.middleware_onion.inject(extradata_to_poa_middleware, layer=0)
+    from web3.middleware import ExtradataToPOAMiddleware
```

Comment:
consistency nit: the error types all start `ExtraData...` and the field in question is `extraData`. What'chu think about changing the middleware name to `ExtraData...`?